src/test/java/edu/cornell/kfs/sys/service/impl/CuUserProcurementProfileValidationServiceImplTest.java

```diff
@@ -16,7 +16,7 @@
 import edu.cornell.kfs.sys.businessobject.UserProcurementProfile;
 import edu.cornell.kfs.sys.service.UserProcurementProfileValidationService;
 
-@ConfigureContext(session = db18)
+@ConfigureContext(session = kad13)
```

Comment:
In the user story, you mentioned that you had to change the user in the session because the old one no longer had the expected access. Should this test add some other KIM setup to make sure the user does indeed have the expected permission or role? Otherwise, we could potentially end up with this situation again in the near future.